includes/functions/plugins/article-status.php

```diff
@@ -99,7 +100,42 @@ function admin_menu() {
  * @return array Modified columns
  */
 function filter_article_columns_article_status( $columns ) {
-	$columns['article_status'] = __( 'Article Status', 'eight-day-week-print-workflow' );
+	$status = array(
+		'post_status' => __( 'Article Status', 'eight-day-week' ),
+	);
+
+			$title_offset = array_search( 'title', array_keys( $columns ) );
```

Comment:
Seems like some extra spacing here

Reply:
@dkotter This was taken care of [here](https://github.com/10up/eight-day-week/blob/950ced938e388276dad7db0cfb1fb8d92e862a0e/includes/functions/plugins/article-status.php#L119).

Reply:
Updated